.quickstart/quickstart.yml

```diff
@@ -8,3 +8,10 @@ destination_configurations:
     dispatch:
       - macro_namespace: dbt_utils
         search_order: [ 'spark_utils', 'dbt_utils' ]
+
+public_models: [
```

Comment:
@fivetran-joemarkiewicz just confirming that the models unavailable in Quickstart shouldn't be included here

Reply:
Correct, only models available in quickstart should be present in the `public_models` config.